casts/spinnerOverlay/src/SpinnerOverlay/index.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { Platform, StyleSheet, View, Modal, TextStyle } from 'react-native';
+import { WaveIndicator } from 'react-native-indicators';
+
+import { UIColor } from '@tonlabs/uikit.core';
+import { UILabel, UILabelColors, UILabelRoles } from '@tonlabs/uikit.themes';
+
+type Props = {
+    visible: boolean;
+    titleContent: string;
+    textContent: string;
+};
+
+const waveConfig = {
+    size: 40,
+    count: 2,
+};
+
+export function UISpinnerOverlay({ visible = false, titleContent, textContent }: Props) {
```

Comment:
I've checked the codebase and noticed that the `titleContent` and `textContent` are not really used. Actually it's used only in UIController in `setSpinnerTitleContent` and `setSpinnerTextContent` methods, which also not used at all anywhere in Surf ;)